lib/dce_lti/controller_methods.rb

```diff
@@ -1,9 +1,7 @@
 module DceLti
   module ControllerMethods
     def authenticate_via_lti
-      if ! current_user
```

Comment:
I reverted the change, but kept the if ! which I replaced with unless (which is the rails way https://github.com/bbatsov/ruby-style-guide#unless-for-negatives)